uvod-do-programovani-2/slicing-metody-moduly/excs/prumerne-teploty.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Průměrné teploty
+demand: 2
+---
+
+Následující tabulka obsahuje průměrné roční teploty v České republice za roky
+2001 až 2010 (zdroj: Český hydrometeorologický ústav).
+
+| rok  | teplota °C |
+| ---- | ---------- |
+| 2001 | 7.8        |
+| 2002 | 8.7        |
+| 2003 | 8.2        |
+| 2004 | 7.8        |
+| 2005 | 7.7        |
+| 2006 | 8.2        |
+| 2007 | 9.1        |
+| 2008 | 8.9        |
+| 2009 | 8.4        |
+| 2010 | 7.2        |
+
+Vytvořte reprezentaci této tabulky pomocí seznamu seznamů. Zde existují dvě možnosti. Nejprve vytvořte seznam, který obsahuje řádky tabulky jako dvouprvkové seznamy a uložte jej do proměnné `radky`. Poté vytvořte seznam, který obsahuje sloupce tabulky, tedy dva seznamy po deseti prvcích. Uložte jej do proměnné `sloupce`.
+
+Pro obě tyto reprezentace vyřešte následující úkoly
+
+1. Získejte teplotu na třetím řádku tabulky.
+1. Získejte rok na pátém řádku tabulky.
+1. Získejte poslední řádek tabulky jako seznam.
+1. Vytvořte tabulku bez prvních dvou řádků.
+1. Vytvořte tabulku pouze z prvních dvou řádků.
+1. Vytvořte tabulku obsahující jen řádky 5, 6, 7, 8.
```

Comment:
Myslí se zde na indexu 5,6,7,8 nebo pátá, šestá, ... řádka? Většinou se na to účastnice ptají.

Reply:
Opraveno.